resources/translations/fi.edn

```diff
@@ -146,7 +147,15 @@
                           :submitted "Haettu"
                           :unknown "Tuntematon"}
                  :submitted "Lähetetty"
-                 :latest-activity "Viimeisin toimenpide"
+                 :todo "Toimia tarvitaan"
+                 :todos {:new-application "Uusi hakemus"
+                         :no-pending-requests "Pyyntöihin on vastattu"
```

Comment:
This could also be "Kaikkiin pyyntöihin on vastattu", it would resolve the ambiguity of "pyyntöihin" being in both definite and indefinite form.